src/main/java/com/github/sh0ckr6/achievementborder/listeners/MobControl.java

```diff
@@ -0,0 +1,107 @@
+package com.github.sh0ckr6.achievementborder.listeners;
+
+import com.destroystokyo.paper.event.entity.EntityAddToWorldEvent;
+import com.destroystokyo.paper.event.entity.EntityRemoveFromWorldEvent;
+import com.github.sh0ckr6.achievementborder.AchievementBorder;
+import org.bukkit.Bukkit;
+import org.bukkit.WorldBorder;
+import org.bukkit.entity.Entity;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+import java.util.logging.Level;
+
+/**
+ * Listener event to control the number of hostile mobs
+ *
+ * @author gtaEPIC
+ * @since 1.1
+ */
+public class MobControl implements Listener {
+    /**
+     * Reference to the plugin
+     *
+     * @since 1.1
+     */
+    private final AchievementBorder plugin;
+
+    private int hostileMobs = 0; // Number of hostile mobs that are spawned
```

Comment:
Convert comment to JavaDoc
```suggestion
    /**
    * Number of hostile mobs that are spawned
    * 
    * @since 1.1
    */
    private int hostileMobs = 0;
```